h/migrations/versions/a2295c2bbe29_make_client_secret_nullable.py

```diff
@@ -0,0 +1,23 @@
+"""
+Make Client.secret nullable
+
+Revision ID: a2295c2bbe29
+Revises: 05bd63575f19
+Create Date: 2017-07-18 11:15:39.885020
+"""
+
+from __future__ import unicode_literals
+
+from alembic import op
+
+
+revision = 'a2295c2bbe29'
+down_revision = '05bd63575f19'
+
+
+def upgrade():
+    op.alter_column('authclient', 'secret', nullable=True)
+
+
+def downgrade():
+    op.alter_column('authclient', 'secret', nullable=False)
```

Comment:
I guess it's better to have the migration blow up (and rollback), instead of doing something automatically that we might not want to have.